[2022-09-10 15:51:59,302][__main__][INFO] - device: cuda:0
[2022-09-10 15:51:59,302][deepke.relation_extraction.standard.tools.preprocess][INFO] - ===== start preprocess data =====
[2022-09-10 15:51:59,303][deepke.relation_extraction.standard.tools.preprocess][INFO] - load raw files...
[2022-09-10 15:51:59,303][utils.ioUtils][INFO] - load csv from /home/zzx/deepke/DeepKE/example/re/standard/data/origin/train.csv
[2022-09-10 15:51:59,305][utils.ioUtils][INFO] - load csv from /home/zzx/deepke/DeepKE/example/re/standard/data/origin/valid.csv
[2022-09-10 15:51:59,307][utils.ioUtils][INFO] - load csv from /home/zzx/deepke/DeepKE/example/re/standard/data/origin/test.csv
[2022-09-10 15:51:59,307][utils.ioUtils][INFO] - load csv from /home/zzx/deepke/DeepKE/example/re/standard/data/origin/relation.csv
[2022-09-10 15:51:59,308][deepke.relation_extraction.standard.tools.preprocess][INFO] - clean data...
Traceback (most recent call last):
  File "run.py", line 51, in main
    preprocess(cfg)
  File "/home/zzx/deepke/DeepKE/src/deepke/relation_extraction/standard/tools/preprocess.py", line 195, in preprocess
    train_data = _clean_data(train_data)
  File "/home/zzx/deepke/DeepKE/src/deepke/relation_extraction/standard/tools/preprocess.py", line 171, in _clean_data
    sent = d['sentence']
KeyError: 'sentence'
Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.